terraform/README.md

```diff
@@ -73,5 +79,42 @@ In the bootstrap module:
 - `providers.tf` lists the required providers
 - `main.tf` sets up s3 bucket to be shared across all environments. It lives in ct-prod to communicate that it should not be deleted
 - `variables.tf` lists the variables that will be needed. Most values are hard-coded in this module
-- `run.sh` Helper script to set up a space deployer and run terraform. The terraform action (`plan`/`apply`/`destroy`) is passed as an argument
+- `run.sh` Helper script to set up a space deployer and run terraform. The terraform action (`show`/`plan`/`apply`/`destroy`) is passed as an argument
 - `teardown_creds.sh` Helper script to remove the space deployer setup as part of `run.sh`
+
+## Bootstrap Process
+
+Bootstrap is used to create an s3 bucket for later terraform runs to store their state in.
+
+### To run the bootstrap the first time or make changes
+
+1. Run `./run.sh plan` to verify that the changes are what you expect
+1. Run `./run.sh apply` to create the shared config bucket and retrieve the bucket credentials
+1. Follow instructions under `Use bootstrap credentials`
+1. Run `./teardown_creds.sh` to remove the space deployer account used to create the s3 bucket
+
+### Retrieving existing bucket credentials
+
+1. Run `./run.sh show` to retrieve the bucket credentials
+1. Follow instructions under `Use bootstrap credentials`
+1. Run `./teardown_creds.sh` to remove the space deployer account used to read the s3 bucket credentials
+
+#### Use bootstrap credentials
+
+1. Add the following to `~/.aws/config`
+
+```
+[profile ct-terraform]
+region = us-gov-west-1
+output = json
+```
+
+2. Add the following to `~/.aws/credentials`
+
+```
+[ct-terraform]
+aws_access_key_id = <<access_key_id from bucket_credentials>>
+aws_secret_access_key = <<secret_access_key from bucket_credentials>>
+```
+
+3. Copy `bucket` from `bucket_credentials` to the backend block of `<env>/providers.tf`
```

Comment:
[question] Will it be obvious where to find these?

Reply:
It should be. `bucket` is labeled exactly as that in the output, same with `access_key_id` and `secret_access_key`